geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/AlterConnectionCommand.java

```diff
@@ -79,8 +80,9 @@ public Result alterConnection(
     // if cc is running, you can only alter connection available in cc service.
     if (ccService != null) {
       // search for the connection that has this id to see if it exists
+      CacheConfig cacheConfig = ccService.getCacheConfig("cluster");
```

Comment:
Throughout, use `InternalClusterConfigurationService.CLUSTER_CONFIG` rather than `"cluster"`.

---

This comment was incorrectly labeled as "outdated," possibly also a result of a force-push update.